README.md

```diff
@@ -47,6 +47,13 @@ Create a Kubernetes cluster with either [Minikube](https://kubernetes.io/docs/ge
 
 If you want to use the Bluemix Container Registry start by [Uploading the images](docs/use-bluemix-container-registry) to the Bluemix Container Registry.
 
+### Deploy to Bluemix
+If you want to deploy the wordpress directly to Bluemix, click on 'Deploy to Bluemix' button below to create a Bluemix DevOps service toolchain and pipeline for deploying the Gitlab sample, else jump to [Steps](##steps)
```

Comment:
This sentence is to long and it has some grammar errors. 

Suggestions:
- "If you want to deploy Wordpress directly to Bluemix, click the 'Deploy to Bluemix' button below.
- The Bluemix DevOps service toolchain and pipeline is pretty wordy and should be deleted. If this is referencing a specific service, then add a hyperlink with the service name as a title.